cluster/examples/kubernetes/ceph/filesystem-test.yaml

```diff
@@ -19,6 +19,7 @@ spec:
         size: 1
         requireSafeReplicaSize: false
   preservePoolsOnDelete: false
```

Comment:
preservePoolsOnDelete can be removed here

Reply:
Done